portfolio/src/main/webapp/script.js

```diff
@@ -78,4 +80,18 @@ function makeComment(heading, content) {
     row.appendChild(td)
 
     return row;
+}
+
+function getDropdownVal() {
+    const dropdown = document.getElementById("num-comments");
+    dropdown.onchange = changeDropdownVal;
+    if (localStorage["num-comments"]) {
+        dropdown.value = localStorage["num-comments"];
+    }   
+}
+
+function changeDropdownVal() {
+    const dropdown = document.getElementById("num-comments");
+    localStorage["num-comments"] = dropdown.value;
+    this.form.submit();
```

Comment:
Are these needed if the html already sets onchange values on the form.

Reply:
I added them because once you reload the page it will always change the value of the dropdown to the first thing in the dropdown. And since you can't select the option that's already selected in the dropdown, it makes it impossible to reselect the first option, so I had to make it so the dropdown value is stored as whatever the user selects.

Reply:
I see.